themes/cardinal_service/lib/scss/opportunities/_opportunities-filter.scss

```diff
@@ -0,0 +1,26 @@
+@charset 'UTF-8';
+
+.opportunity-filter-hero-container {
```

Comment:
This (in addition to the Opportunity page hero styling) can be turned into a universal class.

Reply:
Now once I figure out how to make the hero styling a reusable classes, we can remove this `opportunity_filter.scss` file all together! 🤔